.travis.yml

```diff
@@ -1,24 +1,18 @@
-sudo: required
-dist: trusty
-# Force travis to use its minimal image with default Python settings
+sudo: required 
+dist: trusty 
 language: generic
-compiler:
-  - gcc
-
+notifications:
```

Comment:
>  If travis fails this should be a permanent thread to your mailbox until it is fixed properly

Why? "on_failure: always" -> send mail on every error (default), "on_failure: change" -> send mail only for the first time and after it was resolved(?)